packages/components/pie-button/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { terser } from 'rollup-plugin-terser';
+
+// `npm run build` -> `production` is true
+// `npm run dev` -> `production` is false
+const production = !process.env.ROLLUP_WATCH;
+
+export default {
+	input: './compiled/index.js',
+	output: {
+		name: 'PieButton',
+		file: 'dist/bundle.js',
+		format: 'iife', // immediately-invoked function expression — suitable for <script> tags
+		sourcemap: false
+	},
+	plugins: [
+		resolve(), // tells Rollup how to find date-fns in node_modules
+		commonjs(), // converts date-fns to ES modules
```

Comment:
Are these comments relevant (not sure if we're referencing `date-fns` - looks like a copy/paste thing that needs updating?